hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/Visitor.groovy

```diff
@@ -54,4 +57,7 @@ interface Visitor {
   void visitJob(PigJob job);
   void visitJob(SparkJob job);
   void visitJob(VoldemortBuildPushJob job);
+  void visitJob(HdfsToTeradataJob job);
```

Comment:
We still want to have the visitJob(GobblinJob) here in case we develop other visitors in the future